frameworks/PHP/yii2/app/controllers/SiteController.php

```diff
@@ -57,9 +57,7 @@ public function actionFortunes()
 
         $fortunes[] = $runtimeFortune;
 
-        usort($fortunes, function ($left, $right) {
-            return strcmp($left->message, $right->message);
-        });
+        usort($fortunes, [Fortune::class, 'cmp']);
```

Comment:
I tried both syntaxes in my micro benchmarks, and the difference was negligible. So I chose the cleanest syntax, since `'\\app\\models\\Fortune::cmp'` was ugly, and `'Fortune::cmp'` would fail.

Reply:
Had you tried that fail ?

because that file have:
`use app\models\Fortune;`
So I think that will work.

Reply:
No it won't work, and yes, IIRC, I did confirm the failure myself (but I've since deleted my test file).
From [PHP: Namespaces and dynamic language features](https://www.php.net/namespaces.dynamic) in the official documentation about callables: "One must use the fully qualified name".